data/architecture.yml

```diff
@@ -10,7 +10,7 @@ questions:
     -
         question: 'What is the Symfony mail service transport parameter name?'
         answers:
-            - {value: mailer.transport,   correct: true}
+            - {value: swiftmailer.transport,   correct: true}
```

Comment:
In this question we ask the "service transport *parameter* name", so the actual answer is good.

Anyway, thank you for your report: I only learn something every time a report on this repository is done ^_^